app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/disposaldetails/disposal_fees.scala.html

```diff
@@ -69,26 +71,23 @@
     messages(s"$key$helpTextUserKey$indirectKey.helpText")
 }
 
-@title = @{messages(s"$key$userKey$indirectKey.title")}
 @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+@title = @{ messages(s"$key$userKey$indirectKey.title") }
 
-@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
-
-    @backLinkComponent(backLink)
+@mainTemplate(pageTitle = title, isWelshTranslationAvailable = true, withSignOutLink = true, backLinkUrl = Some(backLink.url)) {
 
     @if(hasErrors) {
         @errorSummary(form, Some(s"$key$userKey$indirectKey"))
     }
 
-    @caption(messages("returns.disposal-details.caption"))
 
-    @formWrapper(controllers.returns.disposaldetails.routes.DisposalDetailsController.whatWereDisposalFeesSubmit(), 'novalidate -> "novalidate") {
+    @formHelper(controllers.returns.disposaldetails.routes.DisposalDetailsController.whatWereDisposalFeesSubmit(), 'novalidate -> "novalidate") {
         @unitInput(
```

Comment:
Can this page use the govuk text input directly instead of the in-service custom component?